### PR TITLE
Add ramen configuration for drenv

### PR DESCRIPTION
**Pull request by nirs**
This is the simplest way to configure *Ramen* assuming that we installed it using `make deploy-*` with drenv based environment such as `regional-dr.yaml`

Issues:
- Does not support installing the bundles and catalog.

I'm not sure yet how we are going to use with the e2e framework, but let's start with a simple working solution and iterate from there.

### CODE REVIEW

**Comment by nirs** [comment]
Change in last version:
- undeploy in reverse order in case resources depend on previous resources.

File: test/ramen-config/deploy [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    --timeout 60s \
+    --context hub
+
+echo "Creating s3 secret in ramen hub namespace"
+kubectl apply -f s3-secret.yaml --context hub
+
+echo "Updating ramen config map"
+# XXX Depends on minikube, will not work for external clusters.
+minio_dr1=$(minikube --profile dr1 -n minio service --url minio)
+minio_dr2=$(minikube --profile dr2 -n minio service --url minio)
```
Comment (nirs) [inline test/ramen-config/deploy:20]: Need to replace minikube with kubectl command - there is an example in https://github.com/RamenDR/ramen/issues/743

**Review by rakeshgm — APPROVED** [review]

**Comment by nirs** [comment]
Changes in last version:
- Document how to configure ramen in the quick start guide
- Improve commit messages (minikube issue was addressed now)

**Review by rakeshgm — APPROVED** [review]

**Comment by rakeshgm** [comment]
> Needs more work - Rakesh found that the wait for status.phase can fail with "status not found". This is the same issue we had in the other scripts, solved by waiting for the status before waiting for the phase.
> 
> I will convert it to python for the next version, using the standard way we wait for status.

I am now able to deploy/configure without any errors. Although, I still see that s3-secrets are not being copied to managed clusters. it can probably be taken care of in the next PR

**Review by ELENAGER — APPROVED** [review]
Looks good for me

**Pull request merged**